deepmd/pt/train/training.py

```diff
@@ -1271,15 +1290,22 @@ def get_single_model(
     return model
 
 
-def get_model_for_wrapper(_model_params):
+def get_model_for_wrapper(
+    _model_params,
+    _loss_params=None,
```

Comment:
very bad design to determine if a model is hessian from the loss function. 
you need to provide a new model type like "ener_hess" for your newly define class EnergyHessianModel

Reply:
plz refer to my reply in another conversation in this script.